frontend/controller/actions/group-kv.js

```diff
@@ -30,14 +30,24 @@ export default (sbp('sbp/selectors/register', {
     console.info('group key-value store data loaded!')
   },
   'gi.actions/group/kv/fetchLastLoggedIn': async ({ contractID }: { contractID: string }) => {
+    const identityContractID = sbp('state/vuex/state').loggedIn?.identityContractID
     const kvData = await sbp('chelonia/kv/get', contractID, KV_KEYS.LAST_LOGGED_IN)
     // kvData could be falsy if the server returns 404
     if (kvData) {
       // Note: this could throw an exception if there's a signature or decryption
       // issue
-      return kvData.data
+      if (kvData.data) {
+        return kvData.data
+      }
+      return {
+        _creator: identityContractID,
+        _reason: 'empty'
+      }
+    }
+    return {
+      _creator: identityContractID,
+      _reason: 'new'
```

Comment:
You're sure this won't break the other KV stuff we have?

Reply:
It shouldn't, it's just adding new keys to it